src/docs/1.4.x/java/resources/gettingstarted/JavaPrereqs.md

```diff
@@ -45,7 +45,7 @@ If you do not have the correct JDK, download it from the  [Oracle website](http:
 
 
 ## Maven
-Lagom requires Maven 3.2.1 or higher, we recommend that you use at least Maven 3.3. In addition to verifying the version on the command line, check that your IDE is using the correct version.
+Lagom requires Maven 3.6.0 or higher. In addition to verifying the version on the command line, check that your IDE is using the correct version.
```

Comment:
Shouldn't this be changed in lagom/lagom? It will be overwritten in the next build.

Reply:
I think the page you want to change in this repo is this one: https://github.com/lagom/lagom.github.io/blob/template/src/main/twirl/com/lightbend/lagom/docs/getstartedjavamaven.scala.html

Reply:
🤦🏼‍♂️